
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a.out is a Catch v2.13.10 host application.
Run with -? for options

-------------------------------------------------------------------------------
Teste vazio
-------------------------------------------------------------------------------
arvore.cpp:98
...............................................................................

arvore.cpp:98: FAILED:
  {Unknown expression after the reported line}
due to a fatal error condition:
  SIGSEGV - Segmentation violation signal

===============================================================================
test cases: 1 | 1 failed
assertions: 2 | 1 passed | 1 failed


ERRORS:
